#goal definition
int32 openLeftDoor
---
#result definition
int32 observation
---
#feedback
string status
